doc/ingest.md

```diff
@@ -26,6 +26,7 @@ type Advertisement struct {
     ContextID Bytes
     Metadata Bytes
     IsRm Bool
+    ExtendedProvider optional ExtendedMetadata
```

Comment:
I wonder how well the field name `ExtendedProvider` with type name `ExtendedMetadata` fit together.

Why the word `Metadata` in the type name?

Reply:
it does encompass the `Metadata` field for the additional provider identities, right?

Reply:
True while encompassing other things like the addrs and peer ID :)

Just now we use the term `Metadata` to refer to a set of bytes that describe retrieval protocol; it has a very specific meaning. But `ExtendedMetadata` is not just `Metadata`; it also includes addresses and peer IDs.

The name `ExtendedProvider` makes sense to me. Because, so far we have used  the term `Provider` to describe Peer ID, addrs and metadata combination. 

Thinking out loud would it make sense to use the same name for both the field and they type? if not it would be nice to avoid terminology overlap.

This is by no means a blocker I just point it out because it'd be hard to change later and might come across as confusing to the reader of docs/schema

Reply:
renamed